domain/host/validation.go

```diff
@@ -42,21 +39,30 @@ func (h *Host) ValidEntity() error {
 	violations.Add(validation.IsIP(h.IPAddr))
 
 	//TODO: what should we be validating here? It doesn't seem to work for
-	glog.V(4).Infof("Validating IPAddr %v for host %s", h.IPAddr, h.ID)
 	ipAddr, err := net.ResolveIPAddr("ip4", h.IPAddr)
 
 	if err != nil {
-		glog.Errorf("Could not resolve: %s to an ip4 address: %v", h.IPAddr, err)
+		plog.WithFields(log.Fields{
+			"hostID": h.ID,
+			"ipAddr": h.IPAddr,
+		}).WithError(err).Error("Could not resolve value to an ip4 address")
```

Comment:
Might this be more appropriate to log at Debug or Warn level, since otherwise it'll be double-logged where ValidEntity() is called?